theme/layout/z-indexes.less

```diff
@@ -0,0 +1,3 @@
+@z-index-header: 3;
+@z-index-apply-button: 2;
```

Comment:
This one doesn't appear to be used internally? This seems too app-specific to be part of the public API.